engines/postgres.py

```diff
@@ -1,3 +1,4 @@
+from past.builtins import basestring
```

Comment:
The usage of `basestring` is not right in this file. It attempts to `decode()` the object, but `basestring` covers both `str` and `bytes`. `str` does not have a `decode` method.